File: coding-platform/training-gate/versi-lama/2d-soal-tambah/nilai-kalimat-matematika.py
def solve(op, A, B):
  opr = ['<', '>', '=']
  math = ['+', '-', '*']
  
  if op in math:
    if op == '-':
      return A - B
    if op == '+':
      return A + B
    if op == '*':
      return A * B
  
  if op in opr:
    if op == '>':
      if A > B:
        return 'benar'
      else:
        return 'salah'
    if op == '<':
      if A < B:
        return 'benar'
      else:
        return 'salah'
    if op == '=':
      if A == B:
        return 'benar'
      else:
        return 'salah'

A, op, B = input().split()
print(solve(op, int(A), int(B)))

File: coding-platform/training-gate/_solusi/2D_-_SOAL_TAMBAHAN/NILAI_KALIMAT_MATEMATIKA.cpp
#include <iostream>
#include <string>
#include <cmath>

using namespace std;

int main()
{
	char op;
	long x, y;
	cin >> x >> op >> y;
	switch (op){
		case '+': cout << x+y; break;
		case '-': cout << x-y; break;
		case '*': cout << x*y; break;
		case '<': (x<y) ? cout << "benar" : cout << "salah"; break;
		case '>': (x>y) ? cout << "benar" : cout << "salah"; break;
		case '=': (x==y) ? cout << "benar" : cout << "salah"; break;
	}
	cout << endl;
	return 0;
}